### PR TITLE
feat(feature-flags): Auto rollback beta

### PR DESCRIPTION
**Pull request by EDsCODE**
## Problem

- add ability to auto rollback feature flags based on sentry errors or insight

## Changes
Screen Shot 2022-10-25 at 10 49 19 PM

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

File: posthog/api/feature_flag.py [hunk 1]
```diff
@@ -172,6 +172,10 @@ def _update_filters(self, validated_data):
         if "get_filters" in validated_data:
             validated_data["filters"] = validated_data.pop("get_filters")
 
+        active = validated_data.get("active", None)
+        if active is True:
+            validated_data["performed_rollback"] = False
```
Comment (neilkakkar) [inline posthog/api/feature_flag.py:177]: 💯

File: posthog/api/feature_flag.py [hunk 2]
```diff
@@ -169,6 +172,10 @@ def _update_filters(self, validated_data):
         if "get_filters" in validated_data:
             validated_data["filters"] = validated_data.pop("get_filters")
 
+        active = validated_data.get("active", None)
+        if active is True:
```
Comment (neilkakkar) [inline posthog/api/feature_flag.py:176]: What language have you been writing in recently? 😋 

Minor nit:

```suggestion
        if active:
```

File: posthog/celery.py [hunk 3]
```diff
@@ -171,6 +171,10 @@ def setup_periodic_tasks(sender: Celery, **kwargs):
             name="count tiles with no filters_hash",
         )
 
+    sender.add_periodic_task(
+        crontab(minute="*"), check_flags_to_rollback.s(), name="check feature flags that should be rolled back"
```
Comment (neilkakkar) [inline posthog/celery.py:175]: Hmm, do we need to still figure out what's going wrong here? Or did you figure it out?

If yes, we should run this less often?

File: posthog/migrations/0274_auto_20221020_1507.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.2.15 on 2022-10-20 15:07
```
Comment (neilkakkar) [inline posthog/migrations/0274_auto_20221020_1507.py:1]: reminder to combine migrations & split PR when we're ready

File: posthog/api/feature_flag.py (hunk 2 above)
Comment (EDsCODE) [reply posthog/api/feature_flag.py:176]: LOL I had this written another way at first like `is not None` and when I did the inverse I explicitly wrote it out...

File: posthog/celery.py (hunk 3 above)
Comment (EDsCODE) [reply posthog/celery.py:175]: Definitely less often

**Comment by EDsCODE** [comment]
test stale

**Pull request closed without merging**